src/caduceus_distillation.py

```diff
@@ -388,20 +404,28 @@ def main() -> None:
     parser.add_argument(
         "--max_train_batches",
         type=int,
-        default=None,
+        # NOTE: this is based on the scaling laws. The teacher was trained on 400k batches, student is about
+        # 15% of the teacher --> 60k batches.
+        default=60_000,
         help="Limit train batches per epoch (defaults to all)",
     )
     parser.add_argument(
         "--max_val_batches",
         type=int,
-        default=10,
-        help="Limit validation batches during training (defaults to 10)",
+        default=128,
+        help="Limit validation batches during training (defaults to 128)",
     )
     parser.add_argument(
         "--max_final_val_batches",
         type=int,
-        default=1000,
-        help="Limit final validation batches (defaults to 1000).",
+        default=1024,
+        help="Limit final validation batches (defaults to 1024).",
+    )
+    parser.add_argument(
+        "--val_check_interval",
+        type=int,
+        default=50,
+        help="Used to compute validation schedule, validation schedule is logarithmic with this interval",
```

Comment:
Questions:
- Any chance we could put the log schedule behind a flag? 
- Any chance we could, if running in log schedule, also log the gradient metrics in log schedule?